ci-operator/step-registry/openshift-logging/enable-cluster-logging/openshift-logging-enable-cluster-logging-commands.sh

```diff
@@ -0,0 +1,582 @@
+#!/bin/bash
+set -o nounset
+
+storage_class=""
+minio_secret=""
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+# setup proxy
+if test -f "${SHARED_DIR}/proxy-conf.sh"
+then
+    source "${SHARED_DIR}/proxy-conf.sh"
+fi
+
+
+### common function ###
+function prepare_namespace(){
+    echo "Prepare namespace $1"
+    if oc get project $1 -o name; then
+        echo "use existing namespace $1"
+    else
+        echo "create namespace $1"
+        oc adm new-project $1  || exit 1
+    fi
+    oc label ns $1 openshift.io/cluster-monitoring="true"
+    oc label ns $1 openshift.io/log-alerting="true"
+}
+
+function are_operators_ready(){
+    echo " Wait the operators ready up to 1 minutes"
+    clo_ready=false
+    lo_ready=false
+    count=1
+    while [[ $count -lt 4 ]]; do
+        if oc -n openshift-logging wait pod --for=condition=Ready  -l name=cluster-logging-operator; then
+            clo_ready=true
+        fi
+        if oc -n openshift-operators-redhat wait pod --for=condition=Ready  -l name=loki-operator-controller-manager; then
+            lo_ready=true
+        fi
+        if [[ $clo_ready == "true" && $lo_ready == "true" ]]; then
+            break
+        fi
+        sleep 15s
+        (( count=count + 1 ))
+    done
+    
+    if [[ $clo_ready == "false" ]]; then
+       echo "show status in openshift-logging "
+       oc -n openshift-logging get sub
+       oc -n openshift-logging get ip
+       oc -n openshift-logging get pod
+       oc -n openshift-operators-redhat get sub
+       oc -n openshift-logging get ip
+    fi
+    
+    if [[ $lo_ready == "false" ]]; then
+       echo "show status in openshift-operators-redhat "
+       oc -n openshift-operators-redhat get sub
+       oc -n openshift-operators-redhat get ip
+       oc -n openshift-operators-redhat get pod
+    fi
+    
+    if [[ $clo_ready == "false" || $lo_ready == "false" ]]; then
+       echo "The openshift-marketplace status"
+       oc -n openshift-marketplace get pod
+       echo "Operator are not ready!"
+       exit 1
+    fi
+}
+
+function get_stroage_class()
+{
+    storage_class=$(oc get storageclass --no-headers=true |cut -d" " -f1 |head -1)
+    if [[ ${storage_class} == "" ]]; then
+        echo "no stroage class for lokistack"
+        exit 1
+    fi 
+    default_class=$(oc get storageclass |grep '(default)' |cut -d" " -f1)
+    if [[ ${storage_class} != "" ]]; then
+        storage_class=${default_class}
+    fi
+}
+
+### check if the cluster is Ok for openshift-logging ###
+function ensure_envs()
+{
+    echo "### Check the step requirement"
+    # exit if the cluster is not ready for cluster-logging
+    if [[ "${KUBECONFIG}"  == "" ]] ; then 
```

Comment:
I think we don't need to do the check as this env should always be set.

Reply:
Dropped.